src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -3045,6 +3045,142 @@ public static void XmlMembersMapping_CompositeType()
         Assert.Equal(requestBodyValue.composite.StringValue, requestBodyActual.composite.StringValue);
     }
 
+    [Fact]
+    public static void Xml_HiddenDerivedFieldTest()
+    {
+        using (var ms = new MemoryStream())
+        {
+            var derived = new DerivedClass { value = "on derived" };
+            var ser = new XmlSerializer(typeof(BaseClass));
+            ser.Serialize(ms, derived);
+
+            ms.Position = 0;
+            var roundtripped = ser.Deserialize(ms) as BaseClass;
```

Comment:
We'd prefer to use SerializeAndDeserialize() method whenever possible, which reduce the code for each test. Can you please have a look at the exiting tests and see if we can use SerializeAndDeserialize for the new tests?